apps/taiga-lumbermill/src/dashboards/crypto/components/swap/swap.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable} from '@angular/core';
+
+interface SwapData {
+    readonly title: string;
+    status: boolean;
+    chosen: string;
```

Comment:
what is status 'false' ? 

avoid to mutate initial data, all data fields should be readonly

Reply:
and remove "choosen" from interface, it's the current selection state not part of the data